### DISCUSSION TITLE
Matlab\Python example for waveguide ports

### DISCUSSION THREAD

**Discussion by gadiLhv** [thliebig/openEMS-Project#101] [General]
I am trying to build some simulation with a waveguide port with a known field distribution.
Is there an example for that? Python or Octave\Matlab are both Ok.

**Comment by thliebig** [comment]
Did you find an example? I think the horn antennas use waveguides as excitation...

**Reply by gadiLhv** [reply]
Yes, thank you.

Cheers!
